services/catalog/service/src/index.ts

```diff
@@ -41,6 +43,16 @@ async function bootstrap(): Promise<void> {
 
   const { readiness } = setupLivenessAndReadiness(config);
 
+  // Check ID service is available
+  if (!(await isServiceAvailable(config.idServiceAuthBaseUrl))) {
+    throw new Error(
+      'The Mosaic Identity Service is required to run this service, but it was not available.',
+    );
+  }
+
+  // Register service health endpoint
+  setupServiceHealthEndpoint(app, config.port);
```

Comment:
think we removed the 2nd arg of the `setupServiceHealthEndpoint`, and only require the `app` object now?

pls ensure mosaic-service-common is the latest RC version that has this new method signature